### PR TITLE
103902 adjustments for initial completion repo

### PR DESCRIPTION
**Pull request by janid1967**
[AB#103902](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/103902)

### CODE REVIEW

File: src/tests/Equinor.ProCoSys.Auth.Tests/Authorization/ClaimsTransformationTests.cs [hunk 1]
```diff
@@ -24,9 +25,12 @@ public class ClaimsTransformationTests
         private readonly string Permission1_Plant1 = "A";
         private readonly string Permission2_Plant1 = "B";
         private readonly string Permission1_Plant2 = "C";
-        private readonly string Project1_Plant1 = "Pro1";
-        private readonly string Project2_Plant1 = "Pro2";
-        private readonly string Project1_Plant2 = "Pro3";
+        private readonly string ProjectName1_Plant1 = "Pro1";
+        private readonly Guid ProjectGuid1_Plant1 = new Guid("11111111-1111-1111-1111-111111111111");
```
Comment (jsoi-eq) [inline src/tests/Equinor.ProCoSys.Auth.Tests/Authorization/ClaimsTransformationTests.cs:29]: Redundant type specification here and in the next couple of lines.

File: src/tests/Equinor.ProCoSys.Auth.Tests/Authorization/ClaimsTransformationTests.cs [hunk 2]
```diff
@@ -53,14 +57,36 @@ public void Setup()
             permissionCacheMock.Setup(p => p.GetPermissionsForUserAsync(Plant1, Oid))
                 .ReturnsAsync(new List<string> {Permission1_Plant1, Permission2_Plant1});
             permissionCacheMock.Setup(p => p.GetProjectsForUserAsync(Plant1, Oid))
-                .ReturnsAsync(new List<string> {Project1_Plant1, Project2_Plant1});
+                .ReturnsAsync(new List<AccessableProject>
+                {
+                    new AccessableProject
```
Comment (jsoi-eq) [inline src/tests/Equinor.ProCoSys.Auth.Tests/Authorization/ClaimsTransformationTests.cs:62]: Redundant type specification

File: src/tests/Equinor.ProCoSys.Auth.Tests/Authorization/ClaimsTransformationTests.cs [hunk 3]
```diff
@@ -53,14 +57,36 @@ public void Setup()
...
+                    {
+                        HasAccess = true, 
+                        Name = ProjectName1_Plant1, 
+                        ProCoSysGuid = ProjectGuid1_Plant1
+                    },
+                    new AccessableProject
+                    {
+                        HasAccess = true,
```
Comment (jsoi-eq) [inline src/tests/Equinor.ProCoSys.Auth.Tests/Authorization/ClaimsTransformationTests.cs:70]: Do you have any tests for when HasAccess is false?

Comment (janid1967) [reply src/tests/Equinor.ProCoSys.Auth.Tests/Authorization/ClaimsTransformationTests.cs:70]: That will be in unit test for GetProjectsForUserAsync
In this context, HasAccess is not significant. I remove initializing HasAccess

**Review by jsoi-eq — APPROVED** [review]

**Pull request merged**